src/routes/tools/streamflow/_constants.js

```diff
@@ -0,0 +1,83 @@
+import layers from "~/helpers/mapbox-layers";
+import { LineAreaChart } from "~/components/tools/Charts";
+import { AvgRange, ModelSummary } from "~/components/tools/Stats";
+import { DEFAULT_STAT_PERIODS } from "../_common/constants";
+
+export const TOOL_SLUG = "streamflow";
+
+export const DEFAULT_STATION_LAYER = layers.find(
+  (d) => d.metadata.title === "Central Valley Subbasins"
+);
+export const DEFAULT_STATION_ID = 1;
+
+export const CLIMATE_VARIABLES = ["streamflow"];
+export const CLIMATE_INDICATORS = [
+  {
+    id: "annual",
+    label: "Annual Total by Months",
+    title: "Annual Total Unimpaired Flows",
+    units: "cfs",
+    decimals: 0,
+    chartComponent: LineAreaChart,
```

Comment:
If `chartComponent` is used by both climate indicators, it probably doesn't need to be a dynamic Svelte component right?

Reply:
removed in https://github.com/berkeley-gif/caladapt-website-2021/pull/187/commits/b29e0dae926eb0023452b6e4f3bdbbd619638f7f